easybuild/easyconfigs/t/Tcl/Tcl-8.6.5-foss-2015b.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'Tcl'
+version = '8.6.5'
+
+homepage = 'http://www.tcl.tk/'
+description = """Tcl (Tool Command Language) is a very powerful but easy to learn dynamic programming language,
+suitable for a very wide range of uses, including web and desktop applications, networking, administration,
+testing and many more."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+source_urls = ["http://prdownloads.sourceforge.net/tcl"]
+sources = ['%(namelower)s%(version)s-src.tar.gz']
+
+dependencies = [
+    ('zlib', '1.2.8'),
+]
+
+configopts = '--enable-threads EXTRA_INSTALL="install-private-headers"'
+
```

Comment:
@nathanhaigh Existing `Tcl` easyconfigs include `runtest = 'test'` to enable running the test suite, did you exclude that here for a particular reason? If not, please include it again (below the `start_dir` line makes sense w.r.t. code style).